wiki/Community/How_you_can_help!/en.md

```diff
@@ -56,14 +64,15 @@ Below are just a few of the ways you can contribute:
 - Check out what's going on in the [Development](https://osu.ppy.sh/community/forums/2) forum; this is where most serious projects happen.
   - Join the [osu!dev Discord](https://discord.gg/ppy)! Even just following along and staying up-to-date with osu! development and projects is a good way to stay informed.
 - Join a [beatmap project](https://osu.ppy.sh/community/forums/53) and learn how fun collaborating with others can be!
-- Start your own project! Whether it's bringing back dancing pippi, or just making something cool for osu!, anything is accepted!
+- Start your own project or join someone elses! Whether it's bringing back dancing pippi, or just making something cool for osu!, anything is accepted!
+    - For a list of list user run projects check out the [projects page](/wiki/Projects)
```

Comment:
> For a list of list

this is a typo, and I'd simplify it further to just `Check out a [list of popular projects](...)`